integration_tests/e2e/prisons/viewSessionTemplateStatus.cy.ts

```diff
@@ -9,34 +9,23 @@ context('Change active/inactive session template', () => {
 
   beforeEach(() => {
     const activePrison = TestData.prisonDto({ active: true })
-    deactivatedSessionTemplate = TestData.sessionTemplate({
-      active: false,
-      prisonId: prisonCode,
-      reference: '-ina.dcc.0f',
-    })
-    activeSessionTemplate = TestData.sessionTemplate({ active: true, prisonId: prisonCode, reference: '-act.dcc.0f' })
+    activeSessionTemplate = TestData.sessionTemplate({ active: true })
```

Comment:
@AledWEvans for info - I struggled a bit to get these session template status tests to work reliably so ended up shuffling stuff around a little. I think the issue was different template references in the different template test objects/stubs